src/delphi/fmx.castlecontrol.reg.pas

```diff
@@ -0,0 +1,47 @@
+{
+  Copyright 2022-2023 Michalis Kamburelis.
+
+  This file is part of "Castle Game Engine".
+
+  "Castle Game Engine" is free software; see the file COPYING.txt,
+  included in this distribution, for details about the copyright.
+
+  "Castle Game Engine" is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+  ----------------------------------------------------------------------------
+}
+
+{ Control with OpenGL context on a Delphi FMX form. }
```

Comment:
Please apply here analogous comments as for VCL counterpart. So

```delphi
{ Design-time functionality for the Castle Game Engine on Delphi FMX form. }
unit Fmx.CastleControl.Design;
```

and remove further `{ Designtime Only }` comments.